stable/traefik/templates/deployment.yaml

```diff
@@ -60,6 +60,16 @@ spec:
           periodSeconds: 10
           successThreshold: 1
           timeoutSeconds: 2
+        {{- if and .Values.acme.enabled .Values.acme.dnsProvider.name }}
+        env:
+        {{- range $k, $_ := (index .Values.acme.dnsProvider .Values.acme.dnsProvider.name) }}
```

Comment:
@krancour is there an agreement on how many versions back of k8s you want to support? Otherwise we could replace this range with `envFrom` that was introduced in k8s 1.6.

Reply:
> @krancour is there an agreement on how many versions back of k8s you want to support?

Hadn't thought about it. Rather than force ourselves to figure that out right now, I'd say leave it as is.